src/lib/isBefore.js

```diff
@@ -1,9 +1,11 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
-  const original = toDate(str);
-  return !!(original && comparison && original < comparison);
+export default function isBefore(date, options) {
+  // accessing 'arguments' for backwards compatibility: isBefore(date [, comparisonDate])
+  // eslint-disable-next-line prefer-rest-params
+  const comparisonDate = (typeof options === 'object' ? options.comparisonDate : arguments[1]) || Date().toString();
```

Comment:
```suggestion
  const comparisonDate = (typeof options === 'object' ? options.comparisonDate : arguments[1]) || Date().toString();

```

Reply:
I don't really see a change in above code suggestion. But it should be irrelevant now anyway :).

Reply:
The change is to have an empty line between `comparisonDate` and `comparison`/`original`. Just a styling suggestion

Reply:
I see, because of the wrapped line. Could make it easier to read, true. Thanks, will fix.